_data/meltano/transformers/dbt-bigquery/dbt-labs.yml

```diff
@@ -66,38 +66,50 @@ settings:
     The path to the `keyfile.json`` to use, if authenticating via service-account method.
 commands:
   clean:
+    name: clean
```

Comment:
@alexmarple - Can you add context on the addition of `name` here? Was this a a temporary workaround to solve the GraphQL challenge?

Reply:
Yeah accessing the key to provide the name was an annoying issue for GraphQL and Gridsome so I just added those as a quick solve.